src/cmd-build

```diff
@@ -310,7 +337,8 @@ cat > tmp/meta.json <<EOF
  "coreos-assembler.image-genver": "${image_genver}",
  "coreos-assembler.image-input-checksum": "${image_input_checksum}",
  "coreos-assembler.code-source": "${src_location}",
- "coreos-assembler.container-config-git": $(jq -M '.git' ${PWD}/coreos-assembler-config-git.json)
+ "coreos-assembler.container-config-git": $(jq -M '.git' ${PWD}/coreos-assembler-config-git.json),
+ "ostree-tarfile-checksum": "${ostree_tarfile_checksum}"
```

Comment:
When I was thinking about implementing this I was thinking about including the ostree tarfile as an entry in `images` since it's just another artifact. WDYT?

Reply:
Yeah, I had the same thought when I wrote this up but went with the easy route. My initial argument was that the tarfile isn't a bootable image in the same sense that the other items under there are, but we do also have `kernel` and `initramfs` in there now. Not sure... I think I'm just hung up on that field being called `images`. :) It probably should've been `artifacts`.

That said, yeah I like that there's already an established convention for `path` and `sha256` in there, and it'd be nice to codify the tarfile name itself. Will switch!

Reply:
Done! :arrow_down: